### PR TITLE
Pretty print + string escape

### PR DESCRIPTION
**Pull request by cy6erGn0m**
Requires PR #6  to be merged before

### CODE REVIEW

**Comment by cy6erGn0m** [comment]
Notice that it is breaking change because if users already handling escapes on their side then it will lead to double escape/unescape. However it is correct behaviour for parser to escape/unescape values

File: src/test/kotlin/com/beust/klaxon/KlaxonTest.kt [hunk 1]
```diff
@@ -55,7 +55,61 @@ class KlaxonTest {
             )
         }
 
-        assertEquals("""{ "1" : null , "2" : 2 } """, j.toJsonString())
+        assertEquals("""{"1":null,"2":2}""", j.toJsonString())
```
Comment (cbeust) [inline src/test/kotlin/com/beust/klaxon/KlaxonTest.kt]: I think the version with spaces looks better, don't you?

Comment (cy6erGn0m) [inline src/test/kotlin/com/beust/klaxon/KlaxonTest.kt]: in pretty-print version there will be spaces but if no pretty-print enabled then it should be as compact as possible

**Comment by cbeust** [comment]
Thanks!

**Pull request merged**